installinator/src/peers.rs

```diff
@@ -124,6 +122,10 @@ impl FetchedArtifact {
         F: FnMut() -> Fut,
         Fut: Future<Output = Result<Peers, DiscoverPeersError>>,
     {
+        // How long to sleep between retries if we fail to find a peer or fail
+        // to fetch an artifact from a found peer.
+        const RETRY_DELAY: Duration = Duration::from_secs(1);
```

Comment:
Let's make this even larger -- maybe 2-5 seconds I think?

Reply:
Whoops, we chatted about this and I forgot to change it. Fixed in af124505